src/Symfony/Component/Yaml/Tests/Fixtures/sfMergeKey.yml

```diff
@@ -43,11 +43,11 @@ yaml: |
 php: |
     array(
         'foo' => array('a' => 'Steve', 'b' => 'Clark', 'c' => 'Brian'),
-        'bar' => array('a' => 'before', 'd' => 'other', 'b' => 'new', 'c' => array('foo' => 'bar', 'bar' => 'foo'), 'x' => 'Oren'),
+        'bar' => array('a' => 'before', 'b' => 'new', 'c' => array('foo' => 'bar', 'bar' => 'foo'), 'd' => 'other', 'x' => 'Oren'),
```

Comment:
This order change is only happening when the merge node is not at the beginning of the mapping. I really think people should not rely on that behavior but I can fix it when using only one node (which should always be the case).